mobile/src/main/java/ch/epfl/sdp/mobile/ui/profile/UserScreen.kt

```diff
@@ -29,15 +33,18 @@ fun UserScreen(
     onMatchClick: (ChessMatch) -> Unit,
     lazyColumnState: LazyListState,
     modifier: Modifier = Modifier,
+    contentPadding: PaddingValues = PaddingValues(),
 ) {
+
   LazyColumn(
       state = lazyColumnState,
       verticalArrangement = Arrangement.Top,
       horizontalAlignment = Alignment.CenterHorizontally,
       modifier = modifier,
+      contentPadding = contentPadding,
   ) {
     item { header() }
     stickyHeader { profileTabBar() }
-    items(matches) { match -> Match(match, { onMatchClick(match) }) }
+    items(matches) { match -> Match(match, ChessIcons.WhiteKing, onMatchClick) }

```

Comment:
Should the color of the white king change whenever I'm the black player or does it stays always white ?

Reply:
No I think it is a default icon on figma